package.json

```diff
@@ -8,7 +8,7 @@
     "url": "git@github.com:mapbox/vtshaver.git"
   },
   "scripts": {
-    "test": "tape test/*.test.js",
+    "test": "nyc --all --include bin --include lib tape test/*.test.js",
```

Comment:
Passing custom `--include` here to avoid `nyc` tracking coverage on JS files inside `mason_packages`.